packages/widgetbook/README.md

```diff
@@ -11,465 +11,4 @@ A flutter package which helps developers cataloguing their widgets, testing them
 <img src="https://raw.githubusercontent.com/widgetbook/widgetbook/main/docs/assets/Screenshot.jpg" alt="Widgetbook Screenshot" />
 </p>
 
```

Comment:
Widgetbook is a flutter package which helps developers cataloguing their widgets, testing them quickly on multiple devices and themes, and sharing them easily with designers and clients. Inspired by Storybook.js.